umm-spec-lib/src/cmr/umm_spec/test/iso19115_expected_conversion.clj

```diff
@@ -51,8 +51,8 @@
  (when online-resource
   (-> online-resource
       (update :Linkage #(url/format-url % true))
-      (update :Name #(su/with-default % true))
-      (update :Description #(str (su/with-default % true) " PublicationReference:")))))
+      (update :MimeType #(when % nil))
```

Comment:
```
(update {:a 1 :b 2} :a #(when % nil))

user=>
{:b 2, :a nil}
(update {:a nil :b 2} :a #(when % nil))

user=>
{:b 2, :a nil}
(update {:b 2} :a #(when % nil))

user=>
{:b 2, :a nil}```

I tried this update, I'm still confused what its trying to do.

Reply:
I understand why you are confused. It took me sometime on this one. I originally just removed MimeType but it didn't  work. It seems that the generated one always have :MimeType, either nil or with value. I'm closing the PR for now because I'm almost done with the second part, will combine them together. I will take another look at this. Thanks.